src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -632,6 +642,22 @@ private string GetLanguagesToken(string mediaInfoLanguages)
             return string.Join("+", tokens.Distinct());
         }
 
+        private void UpdateMediaInfoIfNeeded(string pattern, Dictionary<string, int> minimumMediaInfoSchemaRevisions, EpisodeFile episodeFile, Series series)
+        {
+            var matches = TitleRegex.Matches(pattern);
+
+            var shouldUpdateMediaInfo = matches.Cast<Match>()
+                .Select(m => minimumMediaInfoSchemaRevisions.GetValueOrDefault(m.Value, -1))
+                .Where(r => r > -1)
+                .Any(r => episodeFile.MediaInfo.SchemaRevision < r);
+
+
+            if (shouldUpdateMediaInfo)
+            {
+                _mediaInfoUpdater.Update(episodeFile, series);
```

Comment:
We should honor the EnableMediaInfo config flag that the original UpdateMediaInfoService used. Otherwise the mediainfo might be refreshed even though the user explicitly disabled mediainfo.
Can either add the check here or do it in the UpdateMediaInfoService (effectively ignoring the request for update).

Reply:
Yes no problem